data/examples/declaration/data/unnamed-field-comment-2-out.hs

```diff
@@ -0,0 +1,11 @@
+-- | Describes what sort of dictionary to generate for type class instances
+data Evidence
+  = -- | An existing named instance
+    NamedInstance (Qualified Ident)
+  | -- | Computed instances
```

Comment:
This is a bit insane, but Agda authors do not use Haddock correctly, so too bad.